x/evm/keeper/params.go

```diff
@@ -44,3 +44,7 @@ func (k *Keeper) ChainID(ctx sdk.Context) *big.Int {
 func (k *Keeper) WhitelistedCodehashesBankSend(ctx sdk.Context) []string {
 	return k.GetParams(ctx).WhitelistedCodehashesBankSend
 }
+
+func (k *Keeper) WhitelistedCwCodeHashesForDelegateCall(ctx sdk.Context) [][]byte {
+	return k.GetParams(ctx).WhitelistedCwCodeHashesForDelegateCall
```

Comment:
do we need to add a migration for this? somethingl ike https://github.com/sei-protocol/sei-chain/blob/main/x/dex/migrations/v15_to_v16.go

Reply:
yeah good point.Added